backend/src/main/java/ca/bc/gov/hlth/mohums/controller/UsersController.java

```diff
@@ -269,6 +270,27 @@ private List filterUsersByRole(Optional<String[]> selectedRoles, String clientId
         }
     }
 
+    @GetMapping("/users/{userId}/last-logins")
+    public ResponseEntity<Object> getUserLogins(@PathVariable String userId){
+        int maxRecords = 5000;
+        LocalDate oneYearAgo = LocalDate.now().minus(1, ChronoUnit.YEARS);
```

Comment:
I guess this could just be -> `LocalDate oneYearAgo = LocalDate.now().minusYears(1);`